layouts/default.vue

```diff
@@ -83,4 +83,8 @@ html {
     position: inherit;
   }
 }
+
+p {
+  padding: 0 0 1em 0;
```

Comment:
why not use pr-4 ? for rem. Is it needed to use em here?

Reply:
how would it work with tailwind? I went with pure css by lack of time of checking out tailwind details frankly! 

For rem vs em, I guess rem would be fine since all paragraphs are pretty much same size throughout the site. I thought em is nice because if you have smaller font in some places, you get proportionally smaller padding automatically.

Reply:
I'll roll with this one for now just to get closure on this (and fix the site issues look good asap) but I'm definitely open for css refactors. And, I need to educate myself on tailwind too lol.